src/ashtha/Exam/IndicesOfNumber.java

```diff
@@ -0,0 +1,26 @@
+package ashtha.Exam;
+
+public class IndicesOfNumber {
+
+	int[] getIndicesOfNumber(int[] number, int targetNum) {
+		int[] output = new int[2];
+		for (int index = number.length - 1; index >= 0; index--) {
+			for (int innerIndex = 0; innerIndex < number.length; innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files